internal/cmd/update.go

```diff
@@ -0,0 +1,51 @@
+package cmd
```

Comment:
Do we need this command? If we already show the update message every time the user starts crush I think this might not be needed. Not against having this here but just wondering if it would be better if we just add a `update` command on a followup PR to actually update the binary 🤔